homeassistant/components/light/hue.py

```diff
@@ -98,9 +96,11 @@ def _find_host_from_config(hass, filename=PHUE_CONFIG_FILE):
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Hue lights."""
     # Default needed in case of discovery
-    filename = config.get(CONF_FILENAME)
-    allow_unreachable = config.get(CONF_ALLOW_UNREACHABLE)
-    allow_in_emulated_hue = config.get(CONF_ALLOW_IN_EMULATED_HUE)
+    filename = config.get(CONF_FILENAME, PHUE_CONFIG_FILE)
```

Comment:
Notice how there is a comment to warn us but I still merged it 😢 . Got introduced in https://github.com/home-assistant/home-assistant/pull/5474